src/MumbleProtocol.h

```diff
@@ -89,8 +89,9 @@ namespace Protocol {
 	};
 
 	namespace ReservedTargetIDs {
-		constexpr unsigned int REGULAR_SPEECH  = 0;
-		constexpr unsigned int SERVER_LOOPBACK = 31;
+		constexpr unsigned int REGULAR_SPEECH          = 0;
+		constexpr unsigned int SERVER_LOOPBACK_REGULAR = 30;
```

Comment:
I think the naming "regular" is not very helpful in this case. Maybe `SERVER_LOOPBACK_ADDITIONAL` or something in that direction makes things a bit more expressive? :thinking:

Reply:
``SERVER_LOOPBACK_BROADCAST``?

Reply:
`SERVER_LOOPBACK_AND_BROADCAST` <- that would nicely describe what's going on

Reply:
I actually thought about using "broadcast", but ended up with "regular" because shorter. I agree that the former would be clearer.